docs/bbs/bep.md

```diff
@@ -0,0 +1,31 @@
+# BEP Format
```

Comment:
Can you add a linefeed after all of the titles? GitHub is known to break formatting when titles are present without a linefeed.

Reply:
Alright, seems fair. I already made the changes.